deploy/examples/example-custom-build-images.yaml

```diff
@@ -4,10 +4,6 @@ metadata:
   name: example-quarkus
 spec:
   build:
-    #env:
-    # this value is just an example, set yours (optional)
-    #- name: MAVEN_MIRROR_URL
-    #  value: "http://nexus3.kogito-tools.svc.cluster.local:8081/repository/maven-public/"
```

Comment:
I think we can leave it here as an example for custom environment variables.

Reply:
+1